backend/rates/rates.go

```diff
@@ -318,8 +318,8 @@ func (updater *RateUpdater) updateLast(ctx context.Context) {
 	}
 
 	// Provide conversion rates for testnets as well, useful for testing.
-	for _, testnetUnit := range []string{"TBTC", "RBTC", "TLTC", "GOETH"} {
-		if testnetUnit == "GOETH" {
+	for _, testnetUnit := range []string{"TBTC", "RBTC", "TLTC", "GOETH", "SEPETH"} {
+		if testnetUnit == "GOETH" || testnetUnit == "SEPETH" {
 			rates[testnetUnit] = rates[testnetUnit[2:]]
```

Comment:
also here, for SEPETH you need to cut three chars. i noticed it because in SEPETH accounts the fiat conversions are missing.

Reply:
ah, yeah, makes sense 🤦‍♂️ 
should have done more than search/replace there
though I could swear I had rates in the app when testing it, guess not